login-workflow/src/new-architecture/screens/CreateAccountScreen/CreateAccountScreen.test.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import { cleanup, render, screen, fireEvent, RenderResult } from '@testing-library/react';
+import { CreateAccountScreen } from './CreateAccountScreen';
+import { CreateAccountScreenProps } from './types';
+import {
+    RegistrationContextProvider,
+    RegistrationContextProviderProps,
+    i18nRegistrationInstance,
+} from '../../contexts';
+import { RegistrationWorkflow } from '../../components';
+
+const defaultProps: RegistrationContextProviderProps = {
```

Comment:
I've seen this before but I'm not sure if we really want to reuse a test in other tests. IF we use this in all screen tests it will slow down because jest is calling the tests from registration context provider tests. It does skip the tests but if we configure the 4 or 5 test files like this we might run it to some slowness. I'm good either way but just remember when we have performance problems with slow running tests this might be why....

![image](https://github.com/etn-ccis/blui-react-workflows/assets/119693939/1a3e9b28-7330-4235-afff-9043fc84b0da)

Reply:
Makes sense. Thanks @JeffGreiner-eaton

Reply:
We might need to look at the other full screen tests where the registration context provider defaultProps are used and change them. Notice the time diff running without it.....went from 32 to 15
![image](https://github.com/etn-ccis/blui-react-workflows/assets/119693939/3cc54e21-6dfe-4c02-b5b1-b96c3869e46b)